test/test-coordinator.js

```diff
@@ -316,4 +316,9 @@ TestCoordinator.prototype.getSUTHostPort = function getSUTHostPort() {
     return this.sutHostPort;
 };
 
+TestCoordinator.prototype.checksum = function checksum() {
+    //default to no-checksum. This method is changed based on runtime data of the SUT
+    return 0;
```

Comment:
`0` is a valid checksum value which can get exposed to the test. To make it clear that this is not an actual checksum, but a mock implementation, should we change it to either `-1`, or, if we want to make sure this function is always overwritten, raise an exception?